pkg/daemon/ceph/multus/setup.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package multus
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+)
+
+func Setup() error {
+	logger.Info("determining multus ip")
+	multusIpStr, found := os.LookupEnv(multusIpEnv)
+	if !found {
+		return fmt.Errorf("environment variable %s not set.", multusIpEnv)
+	}
+	logger.Infof("multus ip: %s", multusIpStr)
+
+	logger.Info("determining multus link")
+	multusLinkName, found := os.LookupEnv(multusLinkEnv)
+	if !found {
+		return fmt.Errorf("environment variable %s not set.", multusLinkEnv)
+	}
+	logger.Infof("multus link: %s", multusLinkName)
```

Comment:
Logging is good, but I'm not sure the messages like "determining multus ip" and "determining multus link" are particularly helpful if we are also printing the results of the steps once they are complete.

I think maybe an initial message would be good "setting up multus" or something similar, and then outputting info about the setup results as it goes.